tests/src/python/test_qgsaggregatecalculator.py

```diff
@@ -391,6 +391,11 @@ def testExpression(self):
         self.assertTrue(ok)
         self.assertEqual(val, 5)
 
+        # test with subset
+        val, ok = agg.calculate(QgsAggregateCalculator.Sum, 'fldint', ids=[1, 2])
+        self.assertTrue(ok)
+        self.assertEqual(val, 6.0)
```

Comment:
Can you also add a test with an empty list? (I.e. should be no features used)

Reply:
done